api/oci/extensions/repositories/git/repository.go

```diff
@@ -0,0 +1,114 @@
+package git
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mandelsoft/logging"
+	"github.com/mandelsoft/vfs/pkg/vfs"
+
+	"ocm.software/ocm/api/credentials"
+	cpicredentials "ocm.software/ocm/api/credentials/cpi"
+	"ocm.software/ocm/api/oci/cpi"
+	"ocm.software/ocm/api/oci/extensions/repositories/ctf"
+	"ocm.software/ocm/api/tech/git"
+	"ocm.software/ocm/api/tech/oci/identity"
+	ocmlog "ocm.software/ocm/api/utils/logging"
+)
+
+type Repository interface {
+	cpi.Repository
+}
+
+type RepositoryImpl struct {
+	cpi.RepositoryImplBase
+	logger logging.UnboundLogger
+	spec   *RepositorySpec
+	ctf    *ctf.Repository
+	client git.Client
+}
+
```

Comment:
I dont know what you mean with Implementation framework. I didnt just override creation/open because I believe it would not be enough. I am happy to be corrected here though, would be great if you could give me a more concrete pointer.

Reply:
Misread, wrapped with Repository